.github/ISSUE_TEMPLATE/Feature_Request.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Feature Request
+about: Suggest an idea for the project
+---
+
+<!--
+
+* Filling out this template with the relevant information will help us
+  diagnose and fix the issue. We appreciate the feedback!
```

Comment:
I'd vote for dropping all content from this template since we're not requesting anything beyond a general description.